src/main/java/net/sf/jabref/logic/importer/fetcher/ArXiv.java

```diff
@@ -74,6 +74,7 @@
 
     private static final String API_URL = "http://export.arxiv.org/api/query";
 
+
```

Comment:
Hm, seems like a formatting thing from Eclipse, when I saved the file. They were not intented by me.